packages/rpc-api/src/getBlockCommitment.ts

```diff
@@ -1,14 +1,19 @@
 import type { Lamports, Slot } from '@solana/rpc-types';
 
 type GetBlockCommitmentApiResponse = Readonly<{
+    /**
+     * An array that represents the amount of cluster stake, denominated in {@link Lamports}, that
+     * has voted on the block at each depth from `0` to `MAX_LOCKOUT_HISTORY`.
```

Comment:
The docs presently say ‘`MAX_LOCKOUT_HISTORY` + 1’ but I'm pretty sure that's incorrect. https://github.com/solana-foundation/solana-com/pull/312